eventmesh/ApiServerEventMesh.js

```diff
@@ -0,0 +1,316 @@
+'use strict';
+
+const _ = require('lodash');
+const assert = require('assert');
+const config = require('../common/config');
+const logger = require('../common/logger');
+const CONST = require('../common/constants');
+const EventMeshServer = require('./EventMeshServer');
+const kc = require('kubernetes-client');
+const JSONStream = require('json-stream');
+
+
+const apiserver = new kc.Client({
+  config: {
+    url: 'https://127.0.0.1:9443',
```

Comment:
IP should be the `broker.ip` from settings.

Port can be either a constant or in settigns with default value.